tensorboard/components/tensor_widget/BUILD

```diff
@@ -100,7 +100,6 @@ tf_ts_library(
 
 tf_ts_devserver(
     name = "dev_server",
-    index_html = "demo/demo.html",
```

Comment:
I noticed that removing this was required when I was trying to do this upgrade too. Not blocking but cc @caisq in case this affects the devserver and it needs to be retooled at all to work again after this change.